exercises/Daniel_Murphy/ex02_oo_basics/src/shape.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <cmath>
+#include <algorithm>
+#include <memory>
+#include <map>
+
+class Shape
+{
+private:
```

Comment:
Are you around next week Daniel? There's one last thing I would normally pick up on which is that this private section should be below the public section (just another coding standards thing). 

If you're not around though we can just close this off now without this detail because neither of us will remember what was going on after the holidays.

Exercise one looks good now

Reply:
@DannyHindson Just an FYI, Daniel mentioned on a different PR today was his last day before holidays (in case he doesn't look at this whilst he's enjoying his time off)

Reply:
Correct, I am off next week. That change is rather small so I've made it and I'll just hope it passes the build server. Merry Xmas guys! 🎅